parsing/parser.mly

```diff
@@ -677,7 +692,9 @@ let mk_directive ~loc name arg =
 %token <string> HASHOP
 %token SIG
 %token STAR
-%token <string * string option> STRING
+%token <string * Location.t * string option> STRING
+%token <string * string * Location.t * string option> BRACEPERCENTBRACE
```

Comment:
I dislike the token name. To me a token named `BRACEPERCENTBRACE` looks like `{%{` in program text. You should call it `QUOTED_STRING_EXTENSION` or something.

Reply:
All the other tokens are named after an approximation of the syntax they represent in term of characters, not in term of semantics, but I don't care enough to argue either way.

Reply:
Exactly: a token called `BRACEPERCENTBRACE` should have the text representation `{%{`, not hte whole string. We call the string token `STRING`, not `DOUBLEQUOTE`.

Reply:
Done.